pkg/pillar/scripts/device-steps.sh

```diff
@@ -468,12 +515,15 @@ if [ -f /var/run/watchdog.pid ]; then
 fi
 /usr/sbin/watchdog -c $TMPDIR/watchdogall.conf -F -s &
 
+blockdev --flushbufs "$CONFIGDEV"
 echo "$(date -Ins -u) Initial setup done"
 
-# Print diag output forever on changes
-$BINDIR/diag -c $CURPART -f >/dev/console 2>&1 &
-
 if [ $MEASURE = 1 ]; then
     ping6 -c 3 -w 1000 zedcontrol
     echo "$(date -Ins -u) Measurement done"
 fi
+
+while true; do
+    access_usb
```

Comment:
OK. I get it. The idea is to keep copying diagnostics data to USB at regular internals. Am I right?

Reply:
And also be able to
1) pick up a changed usb.json
2) dump the identity if a USB stick is inserted late. 
#2 is part of a potential new workflow for onboarding where a USB stick is used to pick up the device certificates so that they can be added to the controller (without relying on a product key/serial). Need to validate that workflow.